Packs/CyberArkPAS/Integrations/CyberArkPAS/CyberArkPAS_test.py

```diff
@@ -263,3 +263,227 @@ def test_add_safe_member_permissions_validate(mocker, permission_exist):
     for permission in permissions:
         if 'ManageSafeMembers' in permission.get('Key'):
             assert permission.get('Value') == permission_exist
+
+
+OLD_RESPONSE_SAFES_1 = {
```

Comment:
```suggestion
OLD_RESPONSE_SAFES_1 = {
```

One day, the current version may be considered old. Let's call this `V11_RESPONSE_SAFES` and the other `V12_RESPONSE_SAFES`